documentation/puppet/v3/environment_classes.json

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
Not sure if this helps, but the only interactive JSON schema validator I have any experience with is [jsonschemalint.com](http://jsonschemalint.com/draft4/).